### PR TITLE
listitem all the things

### PR DESCRIPTION
**Pull request by regnisalram**
accidentally worked on two cards:
- listitem design
- when feed item opened, send update request to server

### CODE REVIEW

File: app/src/main/res/drawable/email.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24"
+    android:viewportWidth="24">
+    <path
+        android:fillColor="@color/colorPrimaryDark"
+        android:pathData="M20,8L12,13L4,8V6L12,11L20,6M20,4H4C2.89,4 2,4.89 2,6V18A2,2 0 0,0 4,20H20A2,2 0 0,0 22,18V6C22,4.89 21.1,4 20,4Z" />
+</vector>
```
Comment (zsofinnemeth) [inline app/src/main/res/drawable/email.xml]: please add an empty line

File: app/src/main/res/drawable/email_open.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- drawable/email_open_outline.xml -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="24dp"
+    android:width="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path android:fillColor="#000" android:pathData="M12,15.36L4,10.36V18H20V10.36L12,15.36M4,8L12,13L20,8V8L12,3L4,8V8M22,8V18A2,2 0 0,1 20,20H4A2,2 0 0,1 2,18V8C2,7.27 2.39,6.64 2.97,6.29L12,0.64L21.03,6.29C21.61,6.64 22,7.27 22,8Z" />
+</vector>
```
Comment (zsofinnemeth) [inline app/src/main/res/drawable/email_open.xml]: +line pls

File: app/src/main/res/drawable/rss.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- drawable/rss.xml -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="24dp"
+    android:width="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path android:fillColor="#000" android:pathData="M6.18,15.64A2.18,2.18 0 0,1 8.36,17.82C8.36,19 7.38,20 6.18,20C5,20 4,19 4,17.82A2.18,2.18 0 0,1 6.18,15.64M4,4.44A15.56,15.56 0 0,1 19.56,20H16.73A12.73,12.73 0 0,0 4,7.27V4.44M4,10.1A9.9,9.9 0 0,1 13.9,20H11.07A7.07,7.07 0 0,0 4,12.93V10.1Z" />
+</vector>
```
Comment (zsofinnemeth) [inline app/src/main/res/drawable/rss.xml]: + line pls

**Review by zsofinnemeth — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/email.xml (hunk 1 above)
Comment (regnisalram) [reply app/src/main/res/drawable/email.xml]: done

File: app/src/main/res/drawable/email_open.xml (hunk 2 above)
Comment (regnisalram) [reply app/src/main/res/drawable/email_open.xml]: done

File: app/src/main/res/drawable/rss.xml (hunk 3 above)
Comment (regnisalram) [reply app/src/main/res/drawable/rss.xml]: done

**Review by zsofinnemeth — APPROVED** [review]

**Pull request merged**